src/wasm-interpreter.h

```diff
@@ -249,7 +249,7 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
       return flow;
     }
     NOTE_EVAL1(flow.values);
```

Comment:
Maybe add `Flow::getValues() { return values; }` and use it instead of directly accessing `values` here? Just to be distinguished from `flow.getValue()`.

Reply:
Makes sense. Done!